### PR TITLE
Drop pkg/apis in eventing-contrib.

### PR DESCRIPTION
**Pull request by mattmoor**
This is a more complete fix for #1007 (which is intended for cherry-picking), which pushes the shared functionality into individual packages.

### CODE REVIEW

**Comment by mattmoor** [comment]
/hold

I want to discuss whether we should land #1007 and cherrypick (then land this) or whether I should just redo #1007 against release-0.13.

File: couchdb/source/pkg/apis/sources/v1alpha1/register.go [hunk 1]
```diff
@@ -28,17 +28,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"knative.dev/eventing-contrib/couchdb/source/pkg/apis/sources"
```
Comment (lionelvillard) [inline couchdb/source/pkg/apis/sources/v1alpha1/register.go]: maybe use `"knative.dev/eventing/pkg/apis/sources"`?

File: couchdb/source/pkg/apis/sources/register.go [hunk 2]
```diff
@@ -17,5 +17,5 @@ limitations under the License.
 package sources
 
 const (
-	GroupName = "sources.eventing.knative.dev"
+	GroupName = "sources.knative.dev"
```
Comment (lionelvillard) [inline couchdb/source/pkg/apis/sources/register.go]: we should probably get rid of this file and use the GroupName in `knative.dev/eventing/pkg/apis/sources`

File: github/pkg/apis/sources/register.go [hunk 3]
```diff
@@ -14,5 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package sources contains sources API versions
 package sources
+
+const (
+	GroupName = "sources.knative.dev"
```
Comment (lionelvillard) [inline github/pkg/apis/sources/register.go]: same here.

**Comment by mattmoor** [comment]
Updated.  I partially reverted the first commit, so `group.go` is still there for package comments, but `register.go` is gone and the `v1alpha1` imports now pick up the constant from eventing.

**Comment by lionelvillard** [comment]
thanks!
/lgtm

**Comment by mattmoor** [comment]
@lionelvillard For cherry-picking, I'd be more comfortable with the original (tiny) change.  Are you ok with me preparing that as a CP against release-0.13?

/unhold

**Comment by mattmoor** [comment]
ugh, need to run codegen, sec...

**Comment by lionelvillard** [comment]
yup that's fine with me.

**Comment by mattmoor** [comment]
Alright, RFAL with `Gopkg.lock` in a separate commit 🙏

**Pull request merged**